src/sbl_filing_api/routers/filing.py

```diff
@@ -108,26 +111,33 @@ async def get_submission(request: Request, id: int):
     return JSONResponse(status_code=status.HTTP_204_NO_CONTENT, content=None)
 
 
-@router.put("/institutions/{lei}/filings/{period_code}/submissions/{id}/accept", response_model=SubmissionDTO)
+@router.put("/institutions/{lei}/filings/{period_code}/submissions/{id}/accept", response_model=SubmissionAccepterDTO)
```

Comment:
I think we should just return the updated submission object here.  It will include the accepter stuff.  When we PUT on a Submission, the updated Submission should be returned.  Same if we PUT for a Filing, etc.